accounts/account_manager.go

```diff
@@ -208,3 +219,35 @@ func zeroKey(k *ecdsa.PrivateKey) {
 		b[i] = 0
 	}
 }
+
+func (am *Manager) Export(path string, addr []byte, keyAuth string) error {
+	key, err := am.keyStore.GetKey(addr, keyAuth)
+	if err != nil {
+		return err
+	}
+	return crypto.SaveECDSA(path, key.PrivateKey)
```

Comment:
I've given it some thought and I believe this is actually right. It enforces a permission mode and leaves less room for errors or mistakes. Leave it in.